pkg/cluster/install.go

```diff
@@ -247,11 +247,11 @@ func (m *manager) Install(ctx context.Context) error {
 			steps.Condition(m.operatorConsoleExists, 30*time.Minute, true),
 			steps.Action(m.updateConsoleBranding),
 			steps.Condition(m.operatorConsoleReady, 20*time.Minute, true),
+			steps.Action(m.disableSamples),
+			steps.Action(m.disableOperatorHubSources),
 			steps.Condition(m.clusterVersionReady, 30*time.Minute, true),
 			steps.Condition(m.aroDeploymentReady, 20*time.Minute, true),
 			steps.Action(m.disableUpdates),
```

Comment:
We should also probably move updates up?

Reply:
disableUpdate just removes the channel from the clusterversion CR.  I was mostly focused on things that could impact the clusterversion status on installation.

Reply:
It will speed it up if it doesn't need to pull cincinnati updates

Reply:
updated - ty for the info.  TIL